doc/src/Commands_removed.rst

```diff
@@ -148,6 +148,14 @@ performance characteristics on NVIDIA GPUs. Both, the KOKKOS
 and the :ref:`GPU package <PKG-GPU>` are maintained
 and allow running LAMMPS with GPU acceleration.
 
+i-PI tool
+---------
+
+.. deprecated:: TBD
+
+The i-PI tool has been removed from the LAMMPS distribution.  Instead,
+instructions to install i-PI from PyPi via pip are provided.
+
```

Comment:
If possible I wouldn't indicate it is "deprecated" as we are simply pointing users to a "better" way to access it. I'd say something to the effect of 

```
The i-PI tool is no longer bundled with the LAMMPS package. Users are encouraged to 
install the most recent version from PyPI. Instructions to do so using `pip` are provided.

Reply:
I changed this to ".. versionchanged::"

Reply:
Good with me.